### PR TITLE
Add rules_jvm_external 4.4.1

### PR DESCRIPTION
**Pull request by shs96c**

### CODE REVIEW

**Comment by meteorcloudy** [comment]
Regarding the ".bazelrc" file error, it turns out the [shutil.unpack_archive](https://github.com/bazelbuild/continuous-integration/blob/ae54aa3279218c3dd4a2e15eb7da51562bfcee52/buildkite/bazel-central-registry/bcr_presubmit.py#L209) doesn't support symlink in ".zip" file (because the standard zip format doesn't support symlink, only some customized implementations do), but ".tar.gz" file works fine.

**Comment by shs96c** [comment]
Thank you! I've added https://github.com/bazelbuild/rules_jvm_external/pull/755 to work around this.

File: modules/rules_jvm_external/4.4.2/source.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "integrity": "sha256-r0AZNh398x6Pn37Jmy3P5MsjGJyenuxkA4UDqsxUWTA=",
+    "strip_prefix": "rules_jvm_external-be8d3a0020c565a3668c23da04954dc57a795f19",
+    "url": "https://github.com/bazelbuild/rules_jvm_external/archive/be8d3a0020c565a3668c23da04954dc57a795f19.zip"
```
[the remark this answers is not in the document]
Comment (shs96c) [reply modules/rules_jvm_external/4.4.2/source.json]: Updated. I was iterating to see why the BCR builds were failing without making a full release of `rules_jvm_external`. That's fixed now :)

**Comment by shs96c** [comment]
Builds seem to be failing because a bazel version can't be downloaded, and not because of code in the PR.

**Review by meteorcloudy — APPROVED** [review]
Thank you so much, cannot wait to test it for building Bazel itself.

**Comment by jin** [comment]
\o/

**Pull request merged**